packages/dnb-design-system-portal/gatsby-config.js

```diff
@@ -185,7 +185,7 @@ module.exports = {
     PRESERVE_FILE_DOWNLOAD_CACHE: true,
     PARALLEL_SOURCING: true,
     FAST_DEV: true,
-    DEV_SSR: false,
+    DEV_SSR: true,
```

Comment:
```suggestion
    DEV_SSR: false,
```
This is just here temporary so that others can check out this branch and run it locally, to get the same errors as I've seen.